pages/platform/ai/training_tuto_08_triton_inference_server/guide.en-gb.md

```diff
@@ -0,0 +1,430 @@
+---
+title: AI Training - Tutorial - Get started with NVIDIA Triton Inference Server and AI Training
+slug: training/tuto-triton-inference-server
+excerpt: How to use optimally AI models with NVIDIA Triton Inference Server and AI Training
+section: AI Training - Tutorials
+order: 08
+updated: 2023-05-05
+---
+
+**Last updated 5th May, 2023.**
+
+## Objective
+
+**Triton Inference Server** is part of the NVIDIA AI. Its goal is to streamline and standardize AI inference.
+
+Triton enables users to *deploy*, *run* and *scale* trained AI models from any framework and on any type of resources: **GPU** or **CPU**. It also allows developers to deliver high-performance inference in the Cloud.
+
+The goal of this tutorial is to see how it is possible to deploy **Triton Inference Server** easily thanks to the OVHcloud **AI Training** tool.
+
+![Overview](images/ovh-ai-training-triton-inference-server.png){.thumbnail}
+
+## Requirements
+
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB);
+- [ovhai CLI](https://docs.ovh.com/gb/en/publiccloud/ai/cli/install-client/) installed
+- An AI Training project created inside a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account;
+- A [user for AI Training](https://docs.ovh.com/gb/en/publiccloud/ai/users/);
+- [Docker](https://www.docker.com/get-started) installed on your local computer or on a Virtual Machine;
+- Some knowledge about Docker images and [Dockerfile](https://docs.docker.com/engine/reference/builder/);
+- Some basics on [NVIDIA Triton Inference Server](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/getting_started/quickstart.html)
```

Comment:
I have already seen that in some docs, but maybe it s better to write requirements without the ";"  at the end of sentences ? Don't understand their interest